Source/WebCore/Modules/webxr/WebXROpaqueFramebuffer.h

```diff
@@ -72,12 +73,12 @@ class WebXROpaqueFramebuffer {
     Attributes m_attributes;
     uint32_t m_width { 0 };
     uint32_t m_height { 0 };
-    OwnedGLObject m_depthStencilBuffer;
-    OwnedGLObject m_stencilBuffer;
-    OwnedGLObject m_multisampleColorBuffer;
-    OwnedGLObject m_resolvedFBO;
+    GCGLRenderbuffer m_depthStencilBuffer;
```

Comment:
Nit: I wonder if these names should still have Owned in them. e.g. OwnedGCGLRenderbuffer. Maybe too verbose?

Reply:
`OwnedGLRenderbuffer` was my initial idea but then I decided it was more associated with `GraphicsContextGL` (AKA `GCGL`). I'd be happy with `OwnedGLRenderbuffer`